### ISSUE TITLE
Add New Show skips the step 1 options page

### ISSUE THREAD

**Issue by 007revad** [pymedusa/Medusa#4333]
Video of "Add New Show" skipping step 1 here:
https://www.dropbox.com/s/0v6ry6cps8qy5cf/Medusa_Add_New_Show_Issue.mkv?dl=0

Branch: master
Commit: c65ca2607fae74405dcceaaf8c02f3d29d0313ea
OS: Synology DSM 6.1.7-15284
What you did: Clicked on 'Add Shows > New Show'
What happened: Add New Show dialog skipped past step 1.
What you expected: The option to make selections in step 1.

Logs:
```
2018-06-11 16:53:39 INFO     TORNADO :: [c65ca26] 200 GET /addShows/ (192.168.0.12) 241.76ms
2018-06-11 16:53:40 INFO     TORNADO :: [c65ca26] 200 GET /api/v2/config/main (192.168.0.12) 10.57ms
2018-06-11 16:53:40 INFO     TORNADO :: [c65ca26] 101 GET /ws/ui (192.168.0.12) 3.84ms
2018-06-11 16:53:43 INFO     TORNADO :: [c65ca26] 200 GET /addShows/newShow/ (192.168.0.12) 552.45ms
2018-06-11 16:53:43 INFO     TORNADO :: [c65ca26] 200 GET /js/add-show-options.js?2442 (192.168.0.12) 7.23ms
2018-06-11 16:53:43 INFO     TORNADO :: [c65ca26] 200 GET /js/blackwhite.js?2442 (192.168.0.12) 7.41ms
2018-06-11 16:53:43 INFO     TORNADO :: [c65ca26] 304 GET /api/v2/config/main (192.168.0.12) 5.77ms
2018-06-11 16:53:43 INFO     TORNADO :: [c65ca26] 101 GET /ws/ui (192.168.0.12) 2.62ms
2018-06-11 16:53:48 INFO     TORNADO :: [c65ca26] 200 GET /addShows/ (192.168.0.12) 158.22ms
2018-06-11 16:53:48 INFO     TORNADO :: [c65ca26] 304 GET /api/v2/config/main (192.168.0.12) 10.63ms
2018-06-11 16:53:48 INFO     TORNADO :: [c65ca26] 101 GET /ws/ui (192.168.0.12) 4.46ms
2018-06-11 16:53:50 INFO     TORNADO :: [c65ca26] 200 GET /addShows/newShow/ (192.168.0.12) 145.88ms
2018-06-11 16:53:50 INFO     TORNADO :: [c65ca26] 304 GET /api/v2/config/main (192.168.0.12) 7.37ms
2018-06-11 16:53:50 INFO     TORNADO :: [c65ca26] 101 GET /ws/ui (192.168.0.12) 3.47ms
```

**Comment by medariox** [comment]
Looks correct to me. You are probably looking for `Add Existing Shows`?

**Comment by 007revad** [comment]
Nope, I am trying to use Add New Show. I have 3 top level folders and Medusa now defaults to the first folder in the list. My default folder is the second folder.

This issue only started after I let Medusa update to the latest commits. If I could figure out how to revert back to before latest updates I would - and then ignore future updates.

Add Existing Show is also broken, but in a different way. After clicking on the Add Show button the Add Existing Show page stays open - instead of refreshing and returning to the show list.

**Comment by medariox** [comment]
You can select the folder on the next step.

**Comment by sharkykh** [comment]
> I have 3 top level folders and Medusa now defaults to the first folder in the list. My default folder is the second folder.

Okay, yes, I can confirm this issue. Unfortunately we can't really avoid JavaScript issues popping up.
It is fixed in #4059, so for now you'll have to remove the first root dir and re add it,
so your default root dir is first on the list.

I also don't understand the issue you were trying to show in the video. Everything looks okay to me.
You can search for the show, click Next, verify the root directory, click Next, select show options, and finally click Add Show.

> Add Existing Show is also broken, but in a different way. After clicking on the Add Show button the Add Existing Show page stays open - instead of refreshing and returning to the show list.

Please check the browser's developer console for errors after you click "Add Show",
and post if there are any.

I just tried to do this myself with multiple shows.
If you're not seeing any errors in the console, then looks like an old bug has resurfaced.
Basically, it tries to add the same show twice, for example:
```
Show 1 -> Add Show -> Queued to be added
Show 1 -> Skip Show / Add Show -> ...
Show 2 -> Add Show -> Queued to be added
Show 2 -> Skip Show / Add Show -> ...
etc.
```

**Comment by 007revad** [comment]
Sorry.  I just looked at the 'add new show' pictures on the 'show settings' wiki page (on sickrage and medusa) and realised nothing has changed. So it is actually working.

What made me think there was a missing step was the 0.7ms flash of black screen followed by a 0.7ms flash of the following page: 
![screenshot_2018-06-12 09-45-27](https://user-images.githubusercontent.com/39733752/41263159-7f2ac9de-6e28-11e8-8aeb-389f9c3bea79.png)

I don't know if the page in the above image is an old leftover ui page or a future new ui page. 

So apart from the momentary flash of an old/future page, "add new show" is working okay.

Thanks and sorry for wasting your time.

**Comment by sharkykh** [comment]
I'm not sure what's causing you to see the page malformed like that,
but I am working on making that page more efficient in #4267.
Maybe it will fix these 'flashes'.
